services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/NeutronRouterServiceImpl.java

```diff
@@ -159,6 +159,11 @@ else if (portId == null && subnetId != null) {
         } else {
             return new RouterInterfaceResponse();
         }
+        Router router = this.routerDatabaseService.getByRouterId(routerId);
+        if (router == null) {
+            throw new CanNotFindRouter();
```

Comment:
Recommend to 
1. rename it to RouterUnavailable or RouterNotExist
2. print router id in the exception message

Reply:
Done